app/javascript/components/pages/Header.js

```diff
@@ -13,7 +13,7 @@ function Header(props) {
     return (
         <React.Fragment>
             <nav>
-            {logged_in &&
```

Comment:
We'll add this conditional rendering probably later when we are fine tuning the Links when the user is logged in